guides/Tong/README.md

```diff
@@ -0,0 +1,173 @@
+   
+   ![](associateAdapter3.PNG)
+   
+2. Now boot up PFsense by double-clicking on it from your VirtualBox homepage.
+
+   You will be directed to a disclaimer page after PFsense autoboots. Hit `Enter` to accept.
+
+   ![](accept.png)
+   
+   You will be directed to the installer page. The default selection is `install pfSense`. Hit `Enter` to move forward.
+
+   ![](install.png)
+   
+   Next step is to select filesystem. By default UFS is selected. This option automatically configures the hard drive. Hit `Enter` to continue. 
+   
+   ![](partitioning.png)
+   
+   Now you are on the keymap screen. The default is US standard keyboard map. Hit `Enter` to continue.
+   
+   ![](keymap.png)
+   
+   Select `No` on the next screen when it asks you if you want to open the shell to make anymore manual configurations with the installation. Hit `Enter`.
+   
+   ![](openShellOption.png)
+   
+   Select `Reboot` on the next screen and then hit `Enter`.
+
+   ![](reboot.png)
+   
+3. Take out the PFsense ISO image from the optical disk to avoid going through a loop.
+   
+   After PFsense finishes rebooting, you will be directed to the disclaimer page that you saw after starting up PFsense for the first time. The screens that follow will be the exact same and you'll end up in a loop. The reason for this is because this is the PFsense installer. We have finished installing PFsense and configured the installation from the defaults we selected. This is similar to when booting up a new operating system after inserting the CD into your computer's CD tray. Now, we have to take the "CD" out.
+   
+   PFsense needs to start from the hard disk. To do that, while PFsense is still running, click on the `Devices` tab at the top of the VirtualBox ribbon. Then select `Optical Drives` and `Remove disk from virtual drive`.
+   
+   ![](devicesTab.PNG)
+   
+   Congrats! You finished installing PFsense into VirtualBox! Next, we will cover configuration.
+   
+## Configuring PFsense <a id="configuration"></a>
+
+   
+   Now that the PFsense installer is out, we can go back to VirtualBox and click on our PFsense VM. You will find that you are out of the loop now. Take note that your default login username is `admin` and your password is `pfsense`.
+   
+1. Assign the interfaces.
+
+   On the welcome screen, you will see a list of options available. Type in `1` to choose `option 1`. We are assigning the network interfaces we created earlier into PFsense.
+   
+   ![](option1.png)
+   
+  A list of network interfaces will be shown: em0, em1, em2, and em3. These refer to the network adapters listed on the PFsense `Network` tab: Adapter 1, Adapter 2, and Adapter 3 respectively.
+  
+  ![](lanEm0.png)
+   
+   When the screen asks, `Do VLANs need to be set up first?`, that is referring to setting up the virtual LAN. We have already done that in previous steps, so type `n` for 'no' and hit `Enter`.
```

Comment:
From line 131 to 139. You should consider using the windows snipping tool to include the images for each line. `![](enterInterfaces.PNG)` does visually show the lines 131 to 139 but having a giant paragraph that only explains setup is really off putting. Think about showing an image for each prompt asked.